src/app/components/Home/GroupSelects/index.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useEffect, useState } from "react";
+
+// components
+import {
+  InputItem,
+  LabelForInput,
+  Select,
+  WrapperForSelect,
+} from "./components";
+
+// components
+// import AddressTable from "../AddressTable";
+import ListOfAddresses from "../ListOfAddresses";
+
+// types
+import { OfficesData } from "../../../services/BD/type/OfficesData";
+import { TitleHome } from "../components";
+
+interface IGroupSelector {
+  officesProps: OfficesData;
+}
+
+interface IarrCity {
+  [key: number]: string;
+}
+
+const GroupSelects: React.FC<IGroupSelector> = ({ officesProps }) => {
+  const [country, setCounrty] = useState("");
+  const [arrCity, setArrCity] = useState<Array<IarrCity>>([]);
+  const [city, setCity] = useState("");
+  const [arrAddresses, setArrAddresses] = useState<Array<IarrCity>>([]); 
+
+  useEffect(() => {
+    if (country) {
+      setArrCity(Object.keys(officesProps[country]));
+    }
+    if (city) {
+      setArrAddresses(Object.keys(officesProps[country][city]));
```

Comment:
Addresses have form of `"address" -> "office_id"`. `address` is used in list as text and `office_id` is used in url of each list item. Here you only use `address`. Actually you don't need `arrAddresses`, you can pass `officesProps[country][city]`  or `Object.entries(officesProps[country][city])` to `ListOfAddresses`